Gopkg.lock

```diff
@@ -738,6 +738,7 @@
     "k8s.io/client-go/tools/clientcmd",
     "k8s.io/client-go/tools/remotecommand",
     "k8s.io/client-go/util/exec",
+    "k8s.io/kubernetes/test/utils/image",
```

Comment:
No reason to have this right? I dont know why the solver would need it since you dont reference it and the list is just manually maintained for now?